### PR TITLE
Spelling corrections throughout source

### PR DESCRIPTION
**Pull request by jimmyheaddon**
I figured this was a quick fix so went for it, I can't see any breaking changes.

### CODE REVIEW

File: Source/FakeItEasy.Dictionary.Tests.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- NOTE (adamarlph): For more info see: http://msdn.microsoft.com/en-us/library/bb514188.aspx -->
+<!-- NOTE (adamralph): For more info see: http://msdn.microsoft.com/en-us/library/bb514188.aspx -->
```
Comment (adamralph) [inline Source/FakeItEasy.Dictionary.Tests.xml]: :laughing:

**Comment by adamralph** [comment]
@jimmyheaddon :ok_hand: 

Unfortunately the branch is out of date. Can you rebase onto latest master?

**Comment by jimmyheaddon** [comment]
@adamralph wow you were fast, I was hoping to fix this quickly after I noticed that the "unbearable" change caused a test to fail (ironically).

**Comment by adamralph** [comment]
I'm a loaded gun in the hands of an itchy trigger finger :stuck_out_tongue_winking_eye:

**Comment by jimmyheaddon** [comment]
@adamralph I thought you were US based and wouldn't be up for a while!  I've rebased and have reverted "unbearable" for now, as the test condition will need changing if we correct the spelling :smile:

**Comment by adamralph** [comment]
@jimmyheaddon nope, I'm European through and through :smile: 

Thanks for that, but it looks like you've merged instead of rebasing. See https://github.com/FakeItEasy/FakeItEasy/blob/master/CONTRIBUTING.md#handling-updates-from-upstreammaster

It's probably easiest if you reset the head of your branch back to the previous commit and then follow the rebase procedure, e.g.

* `git checkout staticAnalysis`
* `git reset 12f726cce72dc0f3990a4c8034d924ac6ecde079 --hard`
* ...

Also it would be good if you could squash the two commits into one. That can be done separately, after rebasing.

**Comment by jimmyheaddon** [comment]
Right, 99th time lucky :four_leaf_clover:

**Comment by adamralph** [comment]
Thanks @jimmyheaddon!

**Comment by adamralph** [comment]
Steadily climbing up that leaderboard https://github.com/FakeItEasy/FakeItEasy/graphs/contributors :wink:

**Comment by blairconrad** [comment]
Good stuff, @jimmyheaddon. Thank!
I have to know, though: what tool did you use to detect the errors?

**Comment by jimmyheaddon** [comment]
I use Visual Studio 2015 with ReSharper 9.2 and the ReSpeller Free extension.

There are a bunch of static analysis warnings raised by this and various Roslyn analysers, which I have discussed addressing in #22.

**Comment by blairconrad** [comment]
Ah. Respeller. Cool.

**Pull request merged**